build/source/dshare/popMetadat.f90

```diff
@@ -781,7 +780,7 @@ subroutine read_output_file(err,message)
  outFreq(:) = .false.
```

Comment:
Why is ID removed here?

Reply:
Id was being put into the timestep file and among other variables forcing it to be opened and written by default.

Reply:
It's now part of id_meta, as it's not categorical/'type' data).  id_meta should be expanded later to have gru_ids and hru2gru ids and possibly more, but I'll look into that in another effort.